README.rst

```diff
@@ -227,7 +227,7 @@ Submitting `forms`_:
     $ http -f POST pie.dev/post hello=World
 
 
-See the request that is being sent using one of the `output options`_:
+View the request data sent. Several `output options`_ are supported.
```

Comment:
> View the request being sent using one of the `output options`_:

would work better, because it's not just the request data, it's the whole request.